infra/core/host/appservice.bicep

```diff
@@ -56,23 +56,23 @@ var coreConfig = {
   }
 }
 
-
 var coreConfigWithNetworkRules = union(
   !empty(allowInboundNetworkRange) ? {
     ipSecurityRestrictions: [
-        {
-            ipAddress: '${allowInboundNetworkRange}/32'
-            action: 'Allow'
-            tag: 'Default'
-            priority: 100
-            description: 'Allow specified network range'
-        },{
-            ipAddress: 'Any'
-            action: 'Deny'
-            priority: 2147483647
-            name: 'Deny all'
-            description: 'Deny all access'
-        }
+      {
+        // If allowInboundNetworkRange contains a slash, it's already CIDR notation, otherwise append /32
+        ipAddress: contains(allowInboundNetworkRange, '/') ? allowInboundNetworkRange : '${allowInboundNetworkRange}/32'
```

Comment:
use bicep funcs pls

Reply:
the parseCidr function won't work with a single address, so this approach is fine for IPv4 addresses. For IPv6 you have to specify a different type and the other services need to support it. App Service is the only component in this architecture with this complex IP security restriction setup.